dockstore-webservice/src/main/java/io/dockstore/webservice/core/Language.java

```diff
@@ -0,0 +1,21 @@
+package io.dockstore.webservice.core;
+
+import java.io.Serializable;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+import io.swagger.annotations.ApiModelProperty;
+
+@Embeddable
+public class Language implements Serializable {
+    @Column(name = "language", nullable = false, columnDefinition = "varchar(50)")
+    @ApiModelProperty(value = "The supported language")
+    private String language;
```

Comment:
Thoughts on this being https://github.com/dockstore/dockstore/blob/develop/dockstore-common/src/main/java/io/dockstore/common/DescriptorLanguage.java  or an enum?

Reply:
It's possible

Reply:
Ok, let's look into this before it gets backed into the DB. I'd like some control over the values here to avoid typos, inconsistency and the like. 
Also maybe get rid of https://github.com/dockstore/dockstore/blob/develop/dockstore-common/src/main/java/io/dockstore/common/DescriptorLanguage.java#L74 if not too much of a pain, or spin out if it is.